src/molecules/launch-page/index.tsx

```diff
@@ -1,28 +1,28 @@
-import { useUiConfig } from '../../hook/useConfig'
-import { useColorPalates } from '../../molecules/theme-provider/hooks'
-import styles from './index.module.css'
-
-// import Image from 'next/image';
+import { useUiConfig } from '../../hook/useConfig'; // Updated import statement
+import { useColorPalates } from '../../molecules/theme-provider/hooks';
+import styles from './index.module.css';
+import krushakOdishaImage from './krushak_odisha.png';
```

Comment:
Hey @sarthkjoshi  , the Image should be imported from the config and not hard-coded. Kindly check it out.